src/stories/layouts/Grid/Grid.stories.js

```diff
@@ -0,0 +1,68 @@
+import './styles.scss';
+
+import React from 'react';
+import { Grid } from './Grid';
+
+/**
+ * Example Layout: Grid
+ */
+export default {
+    title: 'Example/Grid',
+    layout: Grid,
+};
+
+const Template = (args) => (
+    <Grid numColumns={args.numColumns}>
+        <div className="grid__item">
+            <h1>Test</h1>
```

Comment:
I would updated the content of the sample grid item to be an `h3` and include a paragraph with sample content, like so:
```
<div className="grid__item">
    <h3>Test</h3>
    <p>Lorem ipsum dolor sit amet, aliquip pertinax intellegebat vel ad. 
    Quo diceret comprehensam id, mazim labore instructior et mea.</p>
</div>
```